xactserver/src/bin/xactserver.rs

```diff
@@ -20,6 +20,12 @@ fn main() -> anyhow::Result<()> {
                 .takes_value(true)
                 .help("ip:port to listen to for connection from other peer servers"),
         )
+        .arg(
+            Arg::with_name("connect-pg")
+                .long("connect-pg")
+                .takes_value(true)
+                .help("ip:port to connect to for sending remote transactions to a PostgreSQL instance")
```

Comment:
"ip:port to connect and send transactions from a remote region to a PostgreSQL instance"